axiom/nr_txn.c

```diff
@@ -1549,6 +1549,7 @@ nr_status_t nr_txn_record_error_worthy(const nrtxn_t* txn, int priority) {
 
 void nr_txn_record_error(nrtxn_t* txn,
                          int priority,
+                         bool add_to_segment,
                          const char* errmsg,
                          const char* errclass,
                          const char* stacktrace_json) {
```

Comment:
Similar feedback to [this comment](https://github.com/newrelic/newrelic-php-agent/pull/767/files#r1437898715) regarding the potential to use specialized wrappers instead of adding bools to each function call.